Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -49,6 +51,32 @@ partial class Controller : ItemComponent, IServerSerializable
         private Item focusTarget;
         private float targetRotation;
 
+        [InGameEditable, Serialize("1", IsPropertySaveable.Yes, description: "The signal sent when the condition is met (if empty, no signal is sent).", alwaysUseInstanceValues: true)]
+        public string Output
+        {
+            get { return output; }
+            set
+            {
+                if (value == null) { return; }
+                output = value;
+                //reactivate (we may not have been previously sending a signal, but might now)
+                IsActive = true;
+            }
+        }
+
+        [InGameEditable, Serialize("0", IsPropertySaveable.Yes, description: "The signal sent when the condition is not met (if empty, no signal is sent).", alwaysUseInstanceValues: true)]
```

Comment:
The description is unclear, what condition? It could also be worth mentioning this does nothing if the `Controller` isn't toggleable, or even better, to hide this altogether if using `ConditionallyEditable`.

Reply:
Clarified the description and mentioned that it only applies if `IsToggle` is true in commit [af157a7](https://github.com/Regalis11/Barotrauma/pull/10419/commits/af157a79551d2add3ae4bbc5050212d39b18e77d).